src/libraries/System.Private.CoreLib/src/System/IO/File.cs

```diff
@@ -632,13 +632,63 @@ public static string ReadAllText(string path, Encoding encoding)
         public static void WriteAllText(string path, string? contents)
             => WriteAllText(path, contents, UTF8NoBOM);
 
+        /// <summary>
+        /// Creates a new file, writes the specified string to the file, and then closes the file.
+        /// If the target file already exists, it is truncated and overwritten.
+        /// </summary>
+        /// <param name="path">The file to write to.</param>
+        /// <param name="contents">The characters to write to the file.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="path"/> is <see langword="null"/>.</exception>
+        /// <exception cref="ArgumentException"><paramref name="path"/> is empty.</exception>
+        /// <exception cref="PathTooLongException">The specified path, file name, or both exceed the system-defined maximum length.</exception>
+        /// <exception cref="DirectoryNotFoundException">The specified path is invalid (for example, it is on an unmapped drive).</exception>
+        /// <exception cref="IOException">An I/O error occurred while opening the file.</exception>
+        /// <exception cref="UnauthorizedAccessException"><paramref name="path"/> specified a file that is read-only.</exception>
+        /// <exception cref="UnauthorizedAccessException"><paramref name="path"/> specified a file that is hidden.</exception>
+        /// <exception cref="UnauthorizedAccessException"><paramref name="path"/> specified a directory.</exception>
+        /// <exception cref="UnauthorizedAccessException">This operation is not supported on the current platform.</exception>
+        /// <exception cref="UnauthorizedAccessException">The caller does not have the required permission.</exception>
```

Comment:
```suggestion
        /// <exception cref="UnauthorizedAccessException"><paramref name="path"/> specified a file that is read-only.
        /// -or- <paramref name="path"/> specified a file that is hidden.
        /// -or- <paramref name="path"/> specified a directory.
        /// -or- This operation is not supported on the current platform.
        /// -or- The caller does not have the required permission.</exception>
```
I haven't seen triple-slash docs declaring the same exception multiple times. I know we've done it [this way](https://github.com/dotnet/runtime/blob/d7ae8c61f0937df4dcffe88ba96eae65fa523a78/src/libraries/System.Private.CoreLib/src/System/IO/File.cs#L162-L164) in several places. cc @carlossanlop

Reply:
I believe this is an acceptable format, and backend tooling generates the "-or-"s.